cypress/integration/types/constants.ts

```diff
@@ -5,12 +5,14 @@ export const clearAllFilters = "Clear all filters";
 export const controls = "Controls";
 export const deleteAction = "Delete";
 export const createNewButton = "Create new";
+export const descriptionFilter = "Description";
```

Comment:
I have put some changes today in this [PR](https://github.com/konveyor/tackle-ui-tests/pull/19)
We will be using these constants in generic way for sorting as well, hence I have renamed the constants to just their string. Example - `export const description = "Description"`

Reply:
Sure. Will wait for PR - #19 to get merged